.gitignore

```diff
@@ -243,3 +243,4 @@ Release/
 /git.VC.db
 *.dSYM
 /contrib/buildsystems/out
+CMakeSettings.json
```

Comment:
Could I ask you to edit the commit message? It currently sounds a bit convoluted, and I think it could be made much clearer. As you are a native speaker, and I am not, I will happily refrain from patronizing you how to improve the wording.

Reply:
Will try. Being a UK native speaker doesn't always make it understandable to all :wink:

Reply:
FWIW I do not believe that the convolutedness had anything UK-specific in it.